Makefile

```diff
@@ -2010,6 +2012,16 @@ ifeq ($(NETWORK_BGP), true)
 netattach: export INTERFACE_BGP_1=${NNCP_BGP_1_INTERFACE}
 netattach: export INTERFACE_BGP_2=${NNCP_BGP_2_INTERFACE}
 endif
+ifeq ($(NETWORK_ISOLATION_IPV4), true)
+netattach: export IPV4_ENABLED=true
+netattach: export CTLPLANE_IPV6_ADDRESS_PREFIX=${NNCP_CTLPLANE_IP_ADDRESS_PREFIX}
```

Comment:
Should this be `*_IP_*` instead of `*_IPV6_*`?

Reply:
oh my, yes it should. Fixed.